tests/discover/distgit.sh

```diff
@@ -32,16 +177,15 @@ rlJournalStart
     rlPhaseStartTest "URL is path to a local distgit repo"
         rlRun -s 'tmt run --remove plans --default \
             discover --how fmf --dist-git-source --dist-git-type fedora --url $tmp/tmt \
-            tests --name tests/prepare/install$'
+            --dist-git-init --dist-git-merge tests --name tests/prepare/install$'
         rlAssertGrep "summary: 1 test selected" $rlRun_LOG -F
     rlPhaseEnd
 
-    # FIXME - use globbing once it is possible (--path tmt-*/tests/execute/framework/data)
     for prefix in "" "/"; do
         rlPhaseStartTest "${prefix}path pointing to the fmf root in the extracted sources"
             rlRun 'pushd tmt'
             rlRun -s "tmt run --remove plans --default discover -v --how fmf \
-            --dist-git-source --ref e2d36db --path ${prefix}tmt-1.7.0/tests/execute/framework/data \
+            --dist-git-source --dist-git-merge --ref e2d36db --dist-git-copy-path ${prefix}tmt-*/tests/execute/framework/data \
```

Comment:
It might work with `path` too however ` --dist-git-copy-path` is more explicit and also allow globbing